mimic/core.py

```diff
@@ -31,7 +35,7 @@ class MimicCore(object):
         that will show up in the Service Catalog
     """
 
-    def __init__(self, clock, apis, domains=()):
+    def __init__(self, clock, apis, domains=(), external_apis=()):
```

Comment:
The `external_apis` parameter should be documented as part of `__init__`'s docstring.